server/playerServer/scripts/PlayerDatabase.js

```diff
@@ -237,6 +237,92 @@ PlayerDatabase.prototype.getMemberLevel = function getMemberLevel (userid, callb
 		callback(err, rows && rows[0] && rows[0].level);
 	});
 };
+PlayerDatabase.prototype.setCoordFavorite = function setCoordFavorite (userid, newX, newY, x, y, name, room, callback) {
+	this.database.query("UPDATE favorites SET x = ?, y = ? WHERE room = ? AND owner = ? AND x = ? AND y = ?",
```

Comment:
Purely as information. The library we use for the database will put the variables in those ? but escaped. This way no sql injections are possible.